mongodbatlas/transition.go

```diff
@@ -0,0 +1,220 @@
+package mongodbatlas
```

Comment:
I understand the intention of this file was to avoid to excessive file changes in the PR, however I would favor removing this file and adjusting imports/references before merging as it currently holds a mix of unrelated functionality.

Reply:
once we agree on file structure, I'd prefer to do it in a different PR that ca be done just after so it's easier to track changes and we avoid to increase this PR more

Reply:
@AgustinBettati I think you're right, I'm changing my mind :-)